src/executors.jl

```diff
@@ -367,7 +367,7 @@ function run_inner_node!(exec::Executor, node::DispatchNode, id::Int)
 
         cond = dispatch!(exec, node)
         debug(logger, "Waiting on $cond")
-        wait(cond)
+        fetch(cond)
```

Comment:
Why doesn't `wait` work here?

Reply:
It may give a deprecation warning on 0.7; I think that's fine though and this should be `wait`